Description: 
   * epswrite changed to eps2write. Fixes the following error: unknown device - epswrite
Author: Prathibha B <prathibhab@cdac.in>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ easychem-0.6/.save.ech
@@ -0,0 +1,3 @@
+# Save file for EasyChem (version 0.6)
+
+smpl 13300 8700 19700 14300 0 0 0 0 1 0 0 0
--- easychem-0.6.orig/export.c
+++ easychem-0.6/export.c
@@ -2084,7 +2084,7 @@ eps_size_export (struct Bond *list, char
   gchar *option;
   int fd, pid, i, j, k, l;
   FILE *file, *in, *out;
-  char *argv[11] = { "gs", "-dNOPAUSE", "-q", "-sDEVICE=epswrite", "-dBATCH",
+  char *argv[11] = { "gs", "-dNOPAUSE", "-q", "-sDEVICE=eps2write", "-dBATCH",
     "-dSAFER", "", "", "-dDEVICEWIDTH=250000", "-dDEVICEHEIGHT=250000", NULL
   };
 
